Makefile

```diff
@@ -101,46 +107,28 @@ mqtt:
 
 define docker_push
 	for svc in $(SERVICES); do \
-		docker push mainflux/$$svc-$(1):$(2); \
+		docker push mainflux/$$svc:$(1); \
 	done
-	docker push mainflux/ui-$(1):$(2)
-	docker push mainflux/mqtt-$(1):$(2)
+	docker push mainflux/ui:$(1)
+	docker push mainflux/mqtt:$(1)
```

Comment:
Hopefully, no one will push arm images instead of amd64. :)

Reply:
This is why I had in the beginning image retagging, but well... Not really needed, maintainers just neet to take care then they are doing the release (and CI is set to push only x86 images).